### DISCUSSION TITLE
Is incompatibility of latest versions with Windows 7 final?

### DISCUSSION THREAD

**Discussion by MMasutin** [Purfview/whisper-standalone-win#364] [Q&A] [answered]
Can specially prepared installers of [Python]https://github.com/adang1345/PythonWin7 help?

**Comment by Purfview** [comment, marked as answer]
>Is incompatibility of latest versions with Windows 7 final?

Yes. https://github.com/Purfview/whisper-standalone-win/blob/f1d3a5fe0bfefa694eb7fda4affbaad1b374a5d1/changelog.txt#L63

>Can specially prepared installers of Python help?

No.

**Comment by MMasutin** [comment]
Is 192.3.4 the last to support 7?

**Reply by Purfview** [reply]
Yes.
